experimental/Schemes/src/ToricBlowups/constructors.jl

```diff
@@ -1,4 +1,5 @@
-function _find_blowup_coordinate_name(X::NormalToricVarietyType, coordinate_name::Union{String, Nothing} = nothing)
+function _find_blowup_coordinate_name(X::NormalToricVarietyType, coordinate_name::Union{Symbol, String, Nothing} = nothing)
+  coordinate_name = coordinate_name isa String ? Symbol(coordinate_name) : coordinate_name
```

Comment:
```suggestion
function _find_blowup_coordinate_name(X::NormalToricVarietyType, _coordinate_name::Union{VarName, Nothing} = nothing)
  coordinate_name = Symbol(_coordinate_name)
```
Some suggestions that again should be applied too all of these cases:
Allow VarName as input (as that is the standard in Oscar). The object before and after type conversion should be saved in variables of different names, as otherwise the compiler has a hard time determining types. Converting a symbol to a symbol gets optimized away, so we can get rid of the branching